### PR TITLE
Add Persistent icon

### LINKED ISSUE #6059: Persistent Company Logo
**Brand Name:** Persistent
**Website:** https://www.persistent.com/
**Alexa rank:** 149,827
  

**Official resources for icon and color:**
image

### PR DESCRIPTION
**Pull request by adsingh14**
Persistent

**Issue:** closes #6059

**Similarweb rank:** 495,248
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8219,6 +8219,12 @@
             "hex": "39457E",
             "source": "https://github.com/tpf/marketing-materials/blob/6765c6fd71bc5b123d6c1a77b86e08cdd6376078/images/onion-logo/tpf-logo-onion.svg"
         },
+        {
+            "title": "Persistent",
+            "hex": "FD5F07",
+            "source": "https://www.persistent.com/wp-content/uploads/2021/03/Persistent-Logo-Kit.zip",
```
Comment (sisp) [inline _data/simple-icons.json:8225]: According to the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), the link to the branding page and not the link to the ZIP archive should be used:

> If the SVG is sourced from:
>
> - **Branding page**: For an SVG from a branding page the source URL should link to the branding page and not the image, PDF, or archive (such as `.zip`) file.

```suggestion
            "source": "https://www.persistent.com/company-overview/branding-guidelines/#nav-persistent-logo",
```

**Review by sisp — CHANGES_REQUESTED** [review]
Thanks for your contribution, @adsingh14! :pray:

I see some notable differences between the original icon and yours:


Could you please give it another try? :pray:

I also have a minor request for change regarding the source URL. Please see my inline comment.

**Comment by adsingh14** [comment]
@sisp > Could you please give it another try?

Is that difference because of precision?

File: _data/simple-icons.json (hunk 1 above)
Comment (sisp) [reply _data/simple-icons.json:8240]: This topic hasn't been resolved.

**Review by sisp — COMMENTED** [review]
The icon looks good now. :tada:



Just the one request regarding the source URL is still open.

**Review by sisp — APPROVED** [review]
LGTM now! :tada: Thanks again, @adsingh14! :pray:

**Comment by sisp** [comment]
Hm, not sure why the checks aren't running. :thinking: Any idea, @simple-icons/maintainers?

**Review by LitoMore — APPROVED** [review]

**Pull request merged**